crates/redis_interface/src/commands.rs

```diff
@@ -852,12 +852,28 @@ impl super::RedisConnectionPool {
             .await
             .change_context(errors::RedisError::ConsumerGroupClaimFailed)
     }
+
+    #[instrument(level = "DEBUG", skip(self))]
+    pub async fn set_keys_using_script(
```

Comment:
this is not a set interface, rather an `incr` interface.

Reply:
Sure will change the name